### PR TITLE
support aggregate function STDDEV_POP()

### LINKED ISSUE #1469: [functions] add std aggregate function
**Summary**

- As mysql: https://dev.mysql.com/doc/refman/8.0/en/aggregate-functions.html#function_std

- Aliases:  `std`, `stddev`, `stddev_pop`

- Doc refer: https://datafuse.rs/development/how-to-write-aggregate-functions/

### PR DESCRIPTION
**Pull request by b41sh**
I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

- Support the aggregation functions `STDDEV_POP`.
- Add test cases and docs.

## Changelog

- New Feature
- Website

## Related Issues

Fixes #1469

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by sundy-li** [comment]
Seems the results are not stable.

File: common/functions/src/aggregates/aggregate_stddev_pop.rs [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    }
+
+    fn state_layout(&self) -> Layout {
+        Layout::new::<AggregateStddevPopState>()
+    }
+
+    fn accumulate(&self, place: StateAddr, arrays: &[Series], _input_rows: usize) -> Result<()> {
+        let state = place.get::<AggregateStddevPopState>();
+        let array: &DFPrimitiveArray<T> = arrays[0].static_cast();
+        for (_row, value) in array.into_no_null_iter().enumerate() {
```
Comment (sundy-li) [inline common/functions/src/aggregates/aggregate_stddev_pop.rs:108]: Null is really annoying, but we have to take it into consideration.

if array.null_count() == 0 {
   // fastpath
} else {
  array.iter().map...
}

Comment (b41sh) [reply common/functions/src/aggregates/aggregate_stddev_pop.rs]: ok, I have modified it. PTAL @sundy-li

**Review by sundy-li — APPROVED** [review]
LGTM, thanks @b41sh

**Pull request merged**